File: apps/web/types/tenant.ts
import { Address } from "./address";

export type Tenant = {
  id: string
  manager_id: string
  fullname: string
  cpf: string
  rg: string
  phone: string
  occupation: string
  marital_status: 'Solteiro(a)' | 'Casado(a)' | 'Amasiado(a)' | 'Divorciado(a)' | 'Viúvo(a)' | 'União Estável',
  address: Address,
};

File: apps/server/internal/application/dto/response/tenant.go
package response

import (
	"imobiliary/internal/domain/types"
)

type TenantDTO struct {
	ID            string              `json:"id"`
	ManagerID     string              `json:"manager_id"`
	Fullname      string              `json:"fullname"`
	CPF           string              `json:"cpf"`
	RG            string              `json:"rg"`
	Phone         string              `json:"phone"`
	Occupation    string              `json:"occupation"`
	MaritalStatus types.MaritalStatus `json:"marital_status"`
	Address       AddressDTO          `json:"address"`
}
